### PR TITLE
EUI-6328-testing

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6328

### Change description ###
Tab redirection issue

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by doganuk — APPROVED** [review]

**Comment by phillip-whitaker-hmcts** [comment]
Happy to review when the build etc is passing. @RiteshHMCTS

File: test/e2e/features/pageObjects/hearingRecordingsCase.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ class HearingRecordingsCase {
       this.continueBtn = element(by.xpath('//button[contains(text(),\'Continue\')]'));
       this.submitBtn = element(by.xpath('//button[contains(text(),\'Submit\')]'));
       this.hearingFilesTabBtn = element(by.xpath('//div[contains(text(),\'Case Hearing Files\')]'));
+      this.hearingFilesTabContainer = $('.mat-tab-body-content .CaseFiles')
```
Comment (phillip-whitaker-hmcts) [inline test/e2e/features/pageObjects/hearingRecordingsCase.js:15]: semicolon

**Review by phillip-whitaker-hmcts — APPROVED** [review]
Just missing a semi colon::::

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**